src/microsim/MSLaneChangerSublane.cpp

```diff
@@ -278,7 +278,12 @@ MSLaneChangerSublane::abortLCManeuver(MSVehicle* vehicle) {
 #endif
         outputLCEnded(vehicle, myCandi, myCandi, priorDirection);
     }
-    vehicle->getLaneChangeModel().setSpeedLat(0);
```

Comment:
not sure why you would put the same line of `setSpeedLat(0) into both branches of if/else

Reply:
The idea is that the Angle does not always have to be computed twice then.
For better readability it could also look like this:
```
double speedLat = vehicle->getLaneChangeModel().getSpeedLat();
vehicle->getLaneChangeModel().setSpeedLat(0);
if (speedLat != 0) {
    // compute new angle here
}
```

Two issues now: 

- the angle will always be computed twice
- The changes at line 824ff. are not in the commit

Reply:
I see your point about double computation but I still don't undertand the `else` branch  since it should be a no-op
```
if (vehicle->getLaneChangeModel().getSpeedLat() != 0) {
   ...
    } else {
        vehicle->getLaneChangeModel().setSpeedLat(0);
    }
}
```

Reply:
The problem of the old code was that if we call `setSpeedLat(0)` first and then call `getSpeedLat()`, SpeedLat will always be 0. That is why I first checked if SpeedLat is 0 and then changed it afterwords (but before the angle computation).